src/app/chat-info/chat-info.component.html

```diff
@@ -1,56 +1,64 @@
-<div class="container dialog-dimension">
+<div class="container">
   <div class="row justify-content-center">
     <div class="col-auto">
-      <img
-        class="groupImage-dimension rounded"
-        src="{{ chatMetaData.groupImageUrl }}"
-      />
+      <div class="group-image my-0">
+        <img src="{{ chatMetaData.groupImageUrl }}" class="img-fluid" />
+      </div>
     </div>
   </div>
-  <div class="row pt-1 justify-content-center">
+  <div class="row justify-content-center">
     <div class="col-auto">
-      <h2 class="m-1">{{ chatMetaData.groupName }}</h2>
+      <h2 class="m-1 text-center">{{ chatMetaData.groupName }}</h2>
     </div>
-    <div class="row pb-2 justify-content-center">
-      <div class="col-auto text-muted">
-        <small>{{ chatMetaData.createdAt | date: "short" }}</small>
-      </div>
+  </div>
+  <div class="row justify-content-center">
+    <div class="col-auto">
+      <small>{{ chatMetaData.createdAt | date: "short" }}</small>
     </div>
   </div>
-  <mat-card class="mb-1">
-    <mat-card-title class="pb-2">About</mat-card-title>
-    <mat-card-content>
-      <p>{{ chatMetaData.groupDescription }}</p>
-    </mat-card-content>
-  </mat-card>
-  <mat-card>
-    <mat-card-title class="pb-2">Members</mat-card-title>
-    <mat-card-content>
-      <div class="row pb-2" *ngFor="let user of chatMembers">
-        <div class="col-auto">
-          <img
-            class="profileImage-dimension rounded-circle"
-            src="{{ user.profileImage }}"
-          />
-        </div>
-        <div class="col-auto align-self-center">
-          <h2 class="m-0">{{ user.displayName }}</h2>
-        </div>
-        <div class="col align-self-center">
-          <button
-            mat-raised-button
-            class="btn float-end"
-            color="primary"
-            (click)="sendDM(user)"
-          >
-            Message
-          </button>
+  <mat-divider></mat-divider>
+  <div class="row pt-1">
+    <div class="col px-0">
+      <h2 class="mb-0">About</h2>
+    </div>
+  </div>
+  <div class="row pb-2 justify-content-center">
+    <div class="col">
+      <mat-card class="group-description">
```

Comment:
The title is outside the card, which causes nonalignment. If we want card, why not utilize it fully? Do we want the card or not?